cpp_polska/thread_local.md

```diff
@@ -0,0 +1,221 @@
+# Mało znany kuzyn statica - thread\_local
+C++11 wprowadził wiele funkcjonalności, bez których nie wyobrażamy sobie pisania nowoczesnego kodu. Należą do nich na przykład move semantics, lambda czy smart pointery. Są też ficzery, z których korzysta się raczej rzadko, ale warto je znać i wiedzieć co potrafią. 
+Jednym z takich dodatków jest słowo kluczowe **thread\_local**.
+
+# Problem
+Ostatnio natrafiłem w aplikacji wielowątkowej na problem z zapisem do bazy danych przy dużym obciążeniu. Fragment kodu odpowiedzialny za komunikację 
+z bazą danych okazał się być nieprzystosowany do pracy na wielu wątkach. Bazą danych było mongo, do połączenia używaliśmy bilbioteki mongocxx. Poniżej
+przykładowy fragment kodu: 
+
+    class MongoCoordinator {
```

Comment:
A jak dodasz "```cpp"
```cpp
class ToKodSieLadnieSformatuje
};
```

Reply:
Formatowanie to sprawa podrzędna, bo na blogu i tak jest inny silnik. Github jest tylko do review :) Wielkie dzięki za wszystkie komentarze!